catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -122,11 +122,29 @@ public UpdateRequest process(UpdateRequest input)
         if ((isValidatorEnforced(validatorName) && validationErrorsExist && enforceErrors)
             || isValidatorEnforced(validatorName) && validationWarningsExist && enforceWarnings) {
           INGEST_LOGGER.debug(
-              "The metacard with id={} is being removed from the operation because it failed the enforced validator [{}].",
+              "The metacard with title={} and id={} is being removed from the operation because it failed the enforced validator [{}].",
+              metacard.getTitle(),
               metacard.getId(),
-              validatorName);
+              validatorName,
+              e);
           return null;
         } else {
+          if (validationErrorsExist) {
+            INGEST_LOGGER.debug(
+                "The metacard with title={} and id={} had an unenforced validation error [{}].",
+                metacard.getTitle(),
+                metacard.getId(),
+                validatorName,
+                e);
+          }
+          if (validationWarningsExist) {
+            INGEST_LOGGER.debug(
+                "The metacard with title={} and id={} had an unenforced validation warning [{}].",
+                metacard.getTitle(),
+                metacard.getId(),
+                validatorName,
+                e);
+          }
```

Comment:
✏️ The "Validation results: {} had warnings and {} had errors." log on line 92 is pretty useless since it doesn't provide any information about the specific metacards. Could we remove that one and add a new INFO log to this line instead (right after the warning and error DEBUG logs) that is worded like `The metacard with title={} and id={} had {} unenforced validation warnings and {} unenforced validation errors.` and pass in the number of warnings and errors respectively?

Reply:
fixed